infra/terraform/README.md

```diff
@@ -201,21 +224,128 @@ $ aws ecs execute-command --cluster h-hash-prod-usea1-ecs --task TASKID --contai
 
 where the `TASKID` is the ID of the Fargate service task. Note that this would start an `sh` session directly into your production container, and should be done with caution (if it ever should be done). It's recommended not to run commands against the production environment like this, and we may at some point restrict access to Fargate Exec.
 
+# Troubleshooting and guides
+
+## What to do if the HASH DB needs **Disaster Recovery**
+
+There's a [recovery playbook](./playbooks/db_recovery.md) on what we should do when the DB is on fire, please follow it to recover a snapshot.
+
+## Help! I need access to the private subnet!
+
+You can access the private subnet using the Bastion host. For more information, please see [the corresponding readme](./hash/bastion.md). You need an SSH privatekey to access the bastion, which you can find in `1password` if you have the appropriate access level.
+
+## How do I get the Terraform state from S3?
+
+Install the AWS CLI and follow instructions to add credentials. Credentials are generated in the IAM User security credentials page. `https://us-east-1.console.aws.amazon.com/iam/home#/users/NAME?section=security_credentials` for a user with the name `NAME`.
+
+Once the CLI has been configured, you should have a `.aws` folder in your home directory. Terraform will automatically pick this up and connect to the S3 Terraform State backend.
+
+## How do I log in to AWS ECR container registry?
+
+If you wish to pull/push container images to ECR, AWS CLI allows you to do so with the following command:
+
+```console
+$ aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin ACCOUNTID.dkr.ecr.us-east-1.amazonaws.com
+..
+```
+
+This will log into the given ECR in the `us-east-1` region.
```

Comment:
This was missing steps in the process to 'pull/push container images to ECR', suggest we just link to the AWS docs which give them all
```suggestion
If you wish to pull/push container images to ECR manually, you must:
1. Login to the registry
1. Build and tag the image
1. Push the image to ECR
Please see the [AWS docs](https://docs.aws.amazon.com/AmazonECR/latest/userguide/docker-push-ecr-image.html) for instructions on how to do so.
```